src/main/java/org/archive/format/text/charset/CharsetDetector.java

```diff
@@ -182,6 +183,8 @@ private static String trimAttrValue(String value) {
 			return value;
 		}
 		String result = value;
+		if (result.isEmpty())
```

Comment:
Yes, you're right. I've first fixed the charset detection in commoncrawl/ia-web-commons, and didn't pay enough attention when transferring the fix ([CharsetDetector](https://github.com/commoncrawl/ia-web-commons/blob/master/src/main/java/org/archive/format/text/charset/CharsetDetector.java) didn't have the "is empty" check on value).

I'll update the pull request. Thanks!